pkg/k8sutil/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package k8sutil simplifies interaction with Kubernetes API.
+package k8sutil
```

Comment:
In my opinion, `pkg/k8sutil` should not be part of Lokomotive Go "API", so I suggest we should move it to `/internal/k8s`, also to avoid using `util` keyword (#290). I mentioned that in #605. I'm not sure if this should be done in this PR though.